sonar-c-plugin/src/main/java/org/sonar/plugins/c/CPlugin.java

```diff
@@ -339,57 +336,73 @@
   private static List<PropertyDefinition> compilerWarningsProperties() {
     String subcateg = "(4) Compiler warnings";
     return new ArrayList<>(Arrays.asList(
-      PropertyDefinition.builder(LANG_PROP_PREFIX + CxxCompilerSensor.REPORT_PATH_KEY)
-        .name("Compiler report(s)")
+      PropertyDefinition.builder(LANG_PROP_PREFIX + CxxCompilerVcSensor.REPORT_PATH_KEY)
```

Comment:
it looks like we don't we miss the following properties now:

```JAVA
  public static final String REPORT_PATH_KEY = "compiler.reportPath";
  public static final String REPORT_CHARSET_DEF = "compiler.charset";
  public static final String PARSER_KEY_DEF = "compiler.parser";
```

as far as I see, they are still used in order to set the project configuration. Is it your intent to hide them?

Reply:
Thx for this hint. They are really missing now.